### ISSUE TITLE
Is Python 3.6.9 really required?

### ISSUE THREAD

**Issue by warrickball** [PlanetHunters/tkmatrix#25]
I recently helped a colleague install `tkmatrix` on some shared systems we have that are still on CentOS 7, which is only distributed with Python 3.6.8. Because of the strict Python requirement in `setup.py`, `tkmatrix` (and `lcbuilder`) isn't installable with `pip` but we appear to have successfully installed it by relaxing these requirements in the source. Can the Python 3.6.9 requirement be relaxed to a lesser version? (I glanced at the [3.6.9 changelog](https://docs.python.org/3.6/whatsnew/changelog.html#python-3-6-9-final) and didn't see any obvious features without which I'd expect problems fitting transit light curves.)

To be more specific, we first upgraded `pip` locally to get version 21.1.3. We then created a virtual environment with `venv` and downloaded and extracted the source tarballs for `tkmatrix` 0.2.3 and `lcbuilder` 0.3.6. For `lcbuilder` then `tkmatrix`, we modified `setup.py` to only require Python >= 3.6.8 (and for `tkmatrix` to only require `lcbuilder>=0.3.4`) and installed each with `python3 -m pip install -e . --user`. I ran the TOI-2257 example for a while without any obvious problems. There were only a few warnings of the form
```
LightkurveDeprecationWarning: The PDCSAP_FLUX function is deprecated and may be removed in a future version.
```

**Comment by martindevora** [comment]
Hi! First of all, thanks for your interest in the package. The only reason we used Python 3.6.9 as the lowest allowed version was because it was the lowest one where we tested MATRIX intensively. I'll review it and if I manage to run the integration tests successfully in a lower Python version I'll publish a new version of MATRIX and will let you know.

In relationship to the warnings you are seeing after your set-up, they are expected right now and it'd seem that the package would be working fine.

Kind regards.

**Comment by martindevora** [comment]
In addition, we are working on adding a docker image to provide a ready-to-use environment with MATRIX, but that only serves if you have `sudo` permissions, which might not be the case.

**Comment by martindevora** [comment]
I've already published the new 0.2.4 version with support for Python 3.6.8 (https://pypi.org/project/tkmatrix/)

Please let me know if it works for you.

**Comment by martindevora** [comment]
I will close the issue as the workaround has been implemented. Please reopen if you consider it doesn't fix your issues.
